benchmarks/matmul_benchmark/README.md

```diff
@@ -0,0 +1,13 @@
+# Matrix Multiplication Benchmark
+
+## Introduction:
+
+This simple benchmark perform multiplication between two matrices reporting the time of execution of the computation.
+
+## Settings:
+
+Through the settings json file it's possible to choose the size of the two matrices. Each setting file contain a couple of values that correspond two the number of row and column of the matrices. So, if the setting file contains the two values "dimension1" and "dimension2", the first matrix will be "dimension1"x"dimension2", while the second will be "dimension2"x"dimension1".  The two dimensions are reported in the setting file name.
+
+## Devices supported:
+
```

Comment:
Add simple example how to run the test (like a command the user can just copy and paste in the shell).

Add also installation instructions of additional packages (pytorch) if needed